server/cmd/mmctl/commands/command.go

```diff
@@ -188,7 +188,9 @@ func createCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error
 func listCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	var teams []*model.Team
 	if len(args) < 1 {
-		teamList, _, err := c.GetAllTeams(context.TODO(), "", 0, 10000)
+		teamList, err := getPages(func(page, numPerPage int, etag string) ([]*model.Team, *model.Response, error) {
+			return c.GetAllTeams(context.TODO(), etag, page, numPerPage)
+		}, 250)
```

Comment:
I wonder if we should define a global const and use it everywhere instead of using a custom number.

Reply:
I think that would be a great idea.

 I am still new to mattermost. Do you know in mattermost where do we generally store global const?
  I am thinking of storing that const `DefaultPageSize` in utils where the getPages function is located what do you think?

Reply:
That sound good to me :+1: